src/cobalt_model/frontmatter.rs

```diff
@@ -449,6 +469,7 @@ pub struct Frontmatter {
     pub description: Option<String>,
     pub excerpt: Option<String>,
     pub categories: Vec<String>,
+    pub tags: Option<Vec<String>>,
```

Comment:
I've chosen non-existent to match our current use of tags, to avoid too much changes from the user (he'll just need to remove `.data` and he's good to go)

Reply:
Hmm, it might be easier to answer this question in the wider space of taxonomies.

For taxonomies, I'm guessing we'll only go off of whats in the frontmatter and not fill in empty arrays for everything in the config.  So we should probably do the same for tags.

Reply:
True as well!

But then, for categories, when migrated to taxonomies, it will change behaviour, won't it?

Reply:
Categories is different enough from tags and other taxonomies that I'm unsure if I'll actually merge it in.
- Categories are hierarchical.  I am unsure if we'll support this in taxonomies.
  - This also ties into permalink support
- Categories are auto-determined from the file path (iirc)

Reply:
True for the hierarchical element! So we're good here :D